index.bs

```diff
@@ -1253,7 +1253,7 @@ Technical Architecture Group Appointments</h5>
 	The [=Team=]'s choice of appointee(s)
 	is subject to ratification by secret ballot
 	by both the [=AB=] and the [=TAG=],
-	each requiring a two-thirds approval.
+	requiring a two-thirds approval from each of these groups, counted separately.
```

Comment:
I think this could be simplified to "from each" without losing the meaning?

Reply:
Simplifying it not quite so far, to `from each of these groups`, [was rejected above](https://github.com/w3c/w3process/pull/837#pullrequestreview-1960870313). I'd be OK with either simplification (though #838 raises more concerns).

Reply:
`from each group.`?